### ISSUE TITLE
Dead project?

### ISSUE THREAD

**Issue by Fierelier** [clamwin/clamwin-py#15]
@sherpya can you give us an update on the state of this project? Issues have been going unanswered and there hasn't been an update on this software in over 2 years. You're participating in other projects, so what's going on?

**Comment by MrCybertux** [comment]
Last update was 3 Years ago now also a lot of important questions are not answered:
https://github.com/clamwin/clamwin/issues/7 <- The official Forum is not usable securely since apparently 2021
https://github.com/clamwin/clamwin/issues/13
https://github.com/clamwin/clamwin/issues/9

Also the software is missing essential components:
https://github.com/clamwin/clamwin/issues/10
![grafik](https://github.com/clamwin/clamwin/assets/67368875/5a1b6eb7-d720-419f-8b22-b6887f3e0692)


And the software never really received much releases for the beginig:
![grafik](https://github.com/clamwin/clamwin/assets/67368875/e4aec9bd-4b1f-499e-9744-20bb0af61a60)

I would argue that this was always a semi dead or not regularly maintained fun project

**Comment by cryptonaut01** [comment]
are there any open-source alternatives for windows these days ?

**Comment by Fierelier** [comment]
@MrCybertux It not automatically scanning is not necessarily an issue for people that are just looking for a simple scan-at-will anti virus. Besides, implementing real time protection likely requires some sort of driver, which is hell to get verified on Windows. The infrequent program updates may not be an issue either, as long as the program is held up to date enough to be able to update and read the definitions file for the viruses, though even this doesn't seem to be the case.

@cryptonaut01 I'd like to know that as well. Up to date and trustworthy anti virus for legacy Windows revisions is hard to come by. At least on supported Windows versions, getting Windows Defender under control may be a good option, even if it can be annoying to maintain with Microsoft constantly changing how that works.

**Comment by MrCybertux** [comment]
@Fierelier 
If automatic scanning is considered a must-have feature to be a fully-featured antivirus, it is a matter of perspective and argumentation. If intended for a non-tech-savvy person, I think yes, but as I said, it depends on the point of view.

The infrequent updates, however, are, at least for me, concerning. Are you comfortable with antivirus software that might have a command injection vulnerability? The shown ClamAV update dates back to February of this year, while the newest version of WinClam is from 2021:
![image](https://github.com/clamwin/clamwin/assets/67368875/acc88ce3-1b62-44bd-a52f-3b923f7b48bf)

and another bug denial of service is not as bad as code execution but still bad enough:
![image](https://github.com/clamwin/clamwin/assets/67368875/d7b3b04c-347d-4542-b6c7-8cccb57c474b)

and another one:
![image](https://github.com/clamwin/clamwin/assets/67368875/389ab46d-c7ff-4786-8adc-9504e264e9af)

And you can find many more such examples, and I think this shows why I believe regular updates are so important, but that could just be my opinion.

If you're simply looking for a scanning tool, have you ever considered the ClamAV Windows package? However, as far as I know, it's only a command-line interface tool.

**Comment by HydraDragonAntivirus** [comment]
Only good and active open source antivirus GUI interface right now is Xylent https://sourceforge.net/projects/xylent/ Which uses ClamAV too

**Comment by MrCybertux** [comment]
> Only good and active open source antivirus GUI interface right now is Xylent https://sourceforge.net/projects/xylent/ Which uses ClamAV too

Nice to see a new Antivirus competitor enter the scene.
But to be honest with you there were a lot of projects like that that started out promising and then vanished in this air like armadito sadly :-(
https://github.com/armadito/armadito-av 

So i would beta test, but since your project was registered on the  2024-02-04  i will wait at least until August 2025 to see if the project is still kicking before recommending you to anyone besides Beta testing :-)


@Fierelier @cryptonaut01 

As far as i see there is no good open source Windows AV now these days.
I prefer open source to but since windows is closed source anyway and the question only maters on Windows versions prior to 10 i would recommend the following for Windows Machines:

0patch for windows https://0patch.com/
Bitdefender Free is suported til 2025 on win 7 
eset online scanner runs fine on older windows and a weekly scan can be set up.


The best option one has in my opinion is use Linux and ClamQt and avoid windows all together

**Comment by HydraDragonAntivirus** [comment]
I now added web heuristics.

**Comment by tretdm** [comment]
Of course, I would like to develop the ClamWin myself. Perhaps I will get to this in the future.
